src/applications/hca/components/ConfirmationPage/ConfirmationPrintView.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+
+const ConfirmationPrintView = ({ name, timestamp }) => {
+  return (
+    <>
+      <img
+        src="/img/design/logo/logo-black-and-white.png"
+        className="vagov-logo vads-u-max-width--100 vads-u-margin-bottom--4"
+        alt=""
```

Comment:
I'm guessing this is only displayed in the print preview? Otherwise I think it'd need an alt value.

Reply:
Correct. This is only for the print view and is decorative only anyway.